src/MySqlConnector/Protocol/Payloads/HandshakeResponse41Payload.cs

```diff
@@ -26,12 +24,16 @@ private static ByteBufferWriter CreateCapabilitiesPayload(ProtocolCapabilities s
 		                          (useCompression ? ProtocolCapabilities.Compress : ProtocolCapabilities.None) |
 		                          ProtocolCapabilities.ConnectionAttributes |
 		                          ProtocolCapabilities.SessionTrack |
-		                          ProtocolCapabilities.DeprecateEof |
+
+		                          // ProtocolCapabilities.DeprecateEof |
```

Comment:
Not sure that this tradeoff is worth the benefit; the EOF packet has been deprecated since MySQL 5.7.5, and this enables `CLIENT_SESSION_TRACK`, etc.